src/main/java/de/numcodex/sq2cql/model/Mapping.java

```diff
@@ -33,31 +33,27 @@ public static Mapping of(TermCode concept, String resourceType) {
     }
 
     @JsonCreator
-    public static Mapping of(@JsonProperty("key") TermCode concept,
+    public static Mapping of(@JsonProperty("key") TermCode key,
                              @JsonProperty("fhirResourceType") String resourceType,
                              @JsonProperty("valueFhirPath") String valueFhirPath,
                              @JsonProperty("fixedCriteria") Modifier... fixedCriteria) {
-        return new Mapping(Objects.requireNonNull(concept), Objects.requireNonNull(resourceType), valueFhirPath,
+        return new Mapping(Objects.requireNonNull(key), Objects.requireNonNull(resourceType), valueFhirPath,
                 fixedCriteria == null ? List.of() : List.of(fixedCriteria));
     }
 
-    public TermCode getConcept() {
-        return concept;
+    public TermCode getKey() {
+        return key;
     }
 
     public String getResourceType() {
         return resourceType;
     }
 
-    public List<Modifier> getFixedCriteria() {
-        return fixedCriteria;
-    }
-
-    public TermCode getTermCode() {
```

Comment:
i think getTermCode is easier to understand as one is later on more interested in the termCode rather than the "key"

Reply:
Same issue as above.